src/components/Error/KuzzleErrorPage.vue

```diff
@@ -11,9 +11,9 @@
     <div class="row message-warning">
       <h5>{{$store.state.kuzzle.errorFromKuzzle.message}}</h5>
       <div class="divider"></div>
-      <div class="message">
+      <div class="message" v-if="errorStatus">
         <i class="fa fa-plug"></i>
-        [{{$store.state.kuzzle.errorFromKuzzle.internal.status}}] {{$store.state.kuzzle.errorFromKuzzle.internal.message}}
+        [{{ errorStatus }}]
```

Comment:
I found that it was redundant with the `$store.state.kuzzle.errorFromKuzzle.message`. My stand is that we should keep just one, but the choice is arbitrary.
Also, I would suggest to drop the `internal.status` code, which IMHO is useless for the user. What do you think?